app/human-detector-app/screens/NotifScreen.tsx

```diff
@@ -31,11 +34,20 @@ export default function NotifScreen({ navigation, route }: Props): React.ReactEl
               }}
             >
               <Text style={styles.dateText}>
-                {notification.timestamp.toLocaleDateString(undefined)}
+                {today.toLocaleDateString(undefined) ===
+                notification.timestamp.toLocaleDateString(undefined)
+                  ? 'Today'
```

Comment:
Is there a better way to check if the date is today?

Reply:
Probably just checking year, date, and month. L

Reply:
man